Dtos/Users/AddUserDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace webapi.Dtos.Users
```

Comment:
make one general dto name as UsersDTO must have all properties in it.

Reply:
Done